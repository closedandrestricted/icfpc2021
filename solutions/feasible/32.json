{"vertices":[[21,18],[14,41],[21,41],[21,19],[0,30]]}