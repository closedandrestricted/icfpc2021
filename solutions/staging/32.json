{"vertices":[[21,14],[3,30],[19,37],[26,29],[5,18]]}