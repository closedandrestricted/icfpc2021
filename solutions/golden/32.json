{"vertices":[[19,17],[20,41],[0,30],[7,22],[21,41]]}